app/constructor/Enemy.js

```diff
@@ -156,9 +159,22 @@ export default class Enemy extends entityPrefab {
     this.idle(true)
     Game.fadeInTween(this)
   }
+
   lootDrop() {
     if (!this.alive) {
       console.log(this.stats.loot[0])
     }
   }
+
+  findClosestPlayer(state) {
+    return state.players.reduce((closestPlayer, player) => {
+      const closestDist = Math.sqrt(Math.pow(closestPlayer.position.x - this.position.x, 2) + Math.pow(closestPlayer.position.y - this.position.y, 2))
```

Comment:
Not super pressing, but finding a way to make this code a little cleaner would be nice.